extensions/Worker.Extensions.EventHubs/EventHubTriggerAttribute.cs

```diff
@@ -30,5 +30,11 @@ public EventHubTriggerAttribute(string eventHubName)
         /// Gets or sets the optional app setting name that contains the Event Hub connection string. If missing, tries to use a registered event hub receiver.
         /// </summary>
         public string? Connection { get; set; }
+
+        /// <summary>
+        /// Gets or sets the configuration to enable batch processing of events. Default value is "true".
+        /// </summary>
+        [DefaultValue(true)]
+        public bool IsBatched { get; set; }
```

Comment:
include default value of "true" here too for backwards compat when we remove the `[DefaultValue]` attribute